### ISSUE TITLE
Speaker + follower + VLNBERT baseline

### ISSUE THREAD

**Issue by guhur** [arjunmajum/vln-bert#9]
Hi Arjun,

I am trying to reproduce your results for your speaker + follower + VLNBERT baseline from your paper.

Can you confirm me that what you did is basically train VLNBERT with the augmented paths from Tan et al.: http://www.cs.unc.edu/~airsplay/aug_paths.json ? 

Have you done any pre-processing step such that joining the augmented paths to create longer paths?

Do you mind sharing the beams and the weights of this baseline?

Thanks,
Pierre

**Comment by arjunmajum** [comment]
We did not use the augmented paths from Tan et al. or any pre-processing (e.g. combining paths) for training VLN-BERT.

The beams we used for pre-training can be downloaded with the `scripts/download-auxilary-data.py` script.

Alternatively, the weights for this baseline can be downloaded from row 4 in the [Model Zoo](https://github.com/arjunmajum/vln-bert#model-zoo).

As we described in the paper, we did combine scores from the speaker and follower models (from Tan, et al.) with the VLN-BERT outputs to get the speaker + follower + VLN-BERT baseline results. To make this step easier I can add the code for combining the outputs (it involves a grid search on the hyper-parameters used to do the combination).
